.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+## Background
+
+## Description of Changes
+
+## Checklist
+
+- [ ] New features are documented
+- [ ] Tests added for bug fixes and new features
+- [ ] (@lanl.gov employees) Update copyright on changed files
```

Comment:
@adamdempsey90 @pdmullen Any thoughts on a default PR template? I wanted to keep it simple, but background and description of changes are both quite useful IMO

Reply:
Simple is good 😄 I also don't think we need a `CHANGELOG.md` as `parth`.

Reply:
If we start putting out releases, we should have a Changelog. It doesn't have to be formatted like parthenon's though.

Reply:
That's a good point about changelogs for release.  I think I would like to move away from the Parthenon approach of requiring every PR to update a living changelog, to some kind of automated system that extracts all the PRs merged into develop from the git commit history and creates a template we can use to write custom release notes, with confidence that we are covering all the major changed features. Just a thought, down the road we should iterate on that.

Reply:
We could collect changelog entries somewhere that describe things that were added, fixed, removed, etc. since the last release. And then at release time collate them into the final Changelog. 

My preference is to separate commit messages from descriptions of changes for users. The latter should be more descriptive and not include things like implementation details / source code stuff.

Reply:
Yeah I agree, commit messages and changelog descriptions for users are pretty different.